Makefile.toml

```diff
@@ -184,7 +184,7 @@ dependencies = ["setup"]
 script_runner = "bash"
```

Comment:
For lack of a better spot to leave a comment: I did an audit of dependencies, checking that everything needed is present and everything present is needed, taking into account the proposed policy of not having Requires in dependencies if they're already in build-dependencies, and the couple special cases.

For unneeded, I found that procps has a build-dependency on ncurses that isn't declared in the .spec file - but it does actually need ncurses, so we should probably fix the .spec file.

For missing, I found kernel Requires filesystem, host-ctr Requires containerd, and ecs-agent Requires docker-engine and iptables, but none of those are listed in Cargo.toml files.

Reply:
> procps has a build-dependency on ncurses that isn't declared in the .spec file - but it does actually need ncurses, so we should probably fix the .spec file.

The only mention I see of ncurses is `--without-ncurses`. Are you sure it's needed?